rpcs3/rpcs3qt/game_compatibility.cpp

```diff
@@ -127,7 +127,7 @@ void game_compatibility::RequestCompatibility(bool online)
 	QNetworkReply* network_reply = m_network_access_manager->get(m_network_request);
 
 	// Show Progress
-	m_progress_dialog.reset(new progress_dialog(tr(".Please wait."), tr("Abort"), 0, 100));
+	m_progress_dialog = new progress_dialog(tr(".Please wait."), tr("Abort"), 0, 100);
```

Comment:
Won't it cause a memory leak when there are multiple requests for the compat database at the same run? I think using unique_ptr was correct.

Reply:
Idk. Doesn't deleteLater take care of that?

Reply:
I didn't see any delete statement which is why I was asking. looking at the code again there's none.

Reply:
Only an explicit call to delete operator deallocates a raw pointer.

Reply:
Dialog::Close, m_pdlg, dialog::deleteLater

Reply:
ahh, seems fine then.